query-engine/core/src/query_graph_builder/extractors/filters/filter_grouping.rs

```diff
@@ -13,9 +13,9 @@ impl FromStr for FilterGrouping {
 
     fn from_str(s: &str) -> QueryGraphBuilderResult<Self> {
         match s.to_lowercase().as_str() {
-            "and" => Ok(Self::And),
-            "or" => Ok(Self::Or),
-            "not" => Ok(Self::Not),
+            filters::AND_LOWERCASE => Ok(Self::And),
```

Comment:
Will think about this a bit more, if we want to have _LOWERCASE or similar in there.

Reply:
We can leave it like you build it for now.